src/FormField/Lookup.php

```diff
@@ -99,69 +117,32 @@ class Lookup extends Input
     public $settings = [];
 
     /**
-     * Default string for presenting filter.
-     *
-     * @var string
-     */
-    public $filterHeaderLabel = 'Filtering options:';
-
-    /**
-     * Default label when no data is selected in filter.
+     * Define callback for generating the row data
+     * If left empty default callback Lookup::defaultRenderRow is used.
      *
-     * @var string
+     * @var null|callable
      */
-    public $filterEmpty = 'All';
-
-    public $filterChain = null;
+    public $renderRowFunction;
 
     /**
-     * Array containing filters.
+     * Whether or not to accept multiple value.
+     *   Multiple values are sent using a string with comma as value delimiter.
```

Comment:
Is it possible to support string here? - if string is set, then use this as a delimiter.